src/public/sass/ITCSS/06_components/pages/certificates/certificate.scss

```
@@ -0,0 +1,91 @@
+@use '../../../02_tools/mixins' as *;
+@use '../../../01_settings/colors' as *;
+@use '../../../01_settings/global' as *;
+@use '../../../01_settings/fonts' as *;
+@use '../../../02_tools/breakpoints' as *;
+.certificate {
+  display: flex;
+  max-width: 9rem;
+
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  padding: 2rem 0.7rem 0 0.7rem;
+  border: 1px solid $color-light-blue;
+  text-decoration: none;
+  transition: border 100ms ease-in-out;
+  transition: box-shadow 500ms ease-in-out;
+
+  &__information {
+    margin: 1rem 0 2rem 0 auto;
+    font-weight: $weight-semi-bold;
+    text-align: center;
+    @include desktop {
+      margin: 3rem 0 2rem 0 auto;
+    }
+  }
+
+  &__name {
+    max-width: 24rem;
+    padding: 0 0.5rem;
+    color: $color-gray;
+    @include fluid-text(0.7rem, 1.8rem, 2vw);
+  }
+
+  &__image-container {
+    margin: auto 0;
+    transition: box-shadow 500ms ease-in-out;
+  }
+  &__image {
+    max-width: 9rem;
+    object-fit: cover;
```

```suggestion
    object-fit: cover;
    transition: all 500ms ease;
```

I think you could add this property in order to have a more smooth presentation.